Mathlib/Algebra/Homology/LocalCohomology.lean

```diff
@@ -206,26 +206,13 @@ def idealPowersToSelfLERadical (J : Ideal R) : ℕᵒᵖ ⥤ SelfLERadical J :=
 
 variable {I J K : Ideal R}
 
-/-- The lemma below essentially says that `idealPowersToSelfLERadical I` is initial in
-`selfLERadicalDiagram I`.
```

Comment:
This part of the docstring was lost.

Reply:
I think this is only relevant in the context of that proof so I removed it.

Reply:
Ah, makes sense.